src/forklift/config.py

```diff
@@ -36,7 +36,7 @@ def create_default_config():
             'configuration': 'Production'
         }
 
-        json_config_file.write(dumps(data))
+        json_config_file.write(dumps(data, sort_keys=True, indent=2, separators=(',', ': ')))
```

Comment:
Aren't we 4 space people? Not a big deal to me. Just wonder if there's a reason.